### PR TITLE
create statusBar component

### PR DESCRIPTION
**Pull request by anson1014**
## Description
Created status bar based on https://www.figma.com/file/UEDqyTnpzM3z95ODhrG2u0/Hacker-Application-Evaluator?node-id=551%3A1610.

image

### CODE REVIEW

File: components/statusBar.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  display: flex;
+`;
+
+const Bar = styled.div`
+  height: 20px;
+  width: ${({ width }) => width}px;
```
Comment (kevin-zou) [inline components/statusBar.js:10]: Would change to 100% width so that the parent container is the one owning the size of this component!

**Review by kevin-zou — APPROVED** [review]
One nit, but otherwise lgtm! 🔥

File: components/statusBar.js (hunk 1 above)
Comment (anson1014) [reply components/statusBar.js]: its been done director

Comment (kevin-zou) [reply components/statusBar.js]: Thank you sir 🚀 merge whenever!

**Pull request merged**